template/src/component/page/Page.tsx

```diff
@@ -0,0 +1,33 @@
+import "./_page.scss";
+
+import React, {useEffect} from "react";
+import classNames from "classnames";
+
+import PageHeader from "./header/PageHeader";
+import PageFooter from "./footer/PageFooter";
+
+interface PageProps {
+  children: React.ReactNode;
+  customClassName?: string;
+  title: string;
+}
+
+function Page({children, customClassName, title}: PageProps) {
+  useEffect(() => {
```

Comment:
Is there any hook for changing document title in other hipo projects. @Hipo/frontend-team 
Imo, this effect can be abstracted into a hook and may look more clean in this way. 💡 

but if you think this is only a bare template and no need for custom hooks etc., that's ok 👍

Reply:
I don't think we have a separate hook file for this, probably because this is a very small effect and doesn't bloat the function body. I believe we can leave it like this, we can consider creating project specific custom hooks if the requirements increase (for example in one project, we had to also update other meta tags, so we separated that logic to a custom hook)